.env

```diff
@@ -0,0 +1,28 @@
+#database connection details
+DB_HOST=localhost
+DB_DRIVER=postgres
+# API_SECRET=98hbun98h  
+DB_USER=assets
+DB_PASSWORD=1111
+DB_NAME=events_radar
+DB_PORT=5432
+
+DB_SOURCE=postgresql://assets:1111@localhost:5432/events_radar?sslmode=disable   
+DB_URL = "host=localhost port=5432 user=assets dbname=events_radar password=1111"
+
+#jwt keys
+USER_KEY = userprivatekey
+ADMIN_KEY = adminprivatekey
+
+
+#port to run
+PORT = 8080
+
+
+#to send varification mail
+SMTP_PORT = 587
+SMTP_HOST = smtp.gmail.com
+SMTP_PASSWORD = yvtltjiqsefafrpz
```

Comment:
Remove the password when pushing publicily, you have to keep that privately. otherwise you try git ignore